prow/apis/prowjobs/v1/types.go

```diff
@@ -350,7 +350,12 @@ func (rac *RerunAuthConfig) IsAllowAnyone() bool {
 }
 
 type ReporterConfig struct {
-	Slack *SlackReporterConfig `json:"slack,omitempty"`
+	ResultStore *ResultStoreReporter `json:"resultstore,omitempty"`
+	Slack       *SlackReporterConfig `json:"slack,omitempty"`
+}
+
+type ResultStoreReporter struct {
+	ProjectID string `json:"project_id,omitempty"`
```

Comment:
nit: We had some confusion previously about which projectID this should actually be (I think there were potentially 3 projectIDs involved) so it would be helpful to name this more precisely or add a comment that describes precisely which projectID this should be set to.

Reply:
Added back documentation.